### PR TITLE
Bug 1383908 Followup edits to PR2969

### PR DESCRIPTION
**Pull request by bfallonf**
#2969

@tpoitras @adellape @ahardin-rh Peer review plz?

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -788,27 +789,20 @@ $ oc delete daemonset logging-fluentd
...
+*Configuring Fluentd to Send Logs to an External Log Aggregator*
 
-Fluentd can be configured to send a copy of its logs to an external log aggregator
-that can understand messages from Fluentd. This allows the ability to send logs
-to data stores other than Elasticsearch and enables the ability to further process
-log records after the {product-title} Fluentd has processed them.
+You can configure Fluentd to send a copy of its logs to an external log
+aggregator, and not the default Elasticsearch, using the `secure-forward`
+plug-in. From there, you can further process log records after the
+{product-title} Fluentd has processed them.
```
Comment (tpoitras) [inline install_config/aggregate_logging.adoc]: "after the {product-title} Fluentd has processed them" feels awkward. Should the word "hosted" go between the product-title macro and Fluentd? Something else perhaps?

**Comment by tpoitras** [comment]
@bfallonf just one nit from me, otherwise LGTM :seat: :seedling: :see_no_evil:

**Comment by bfallonf** [comment]
Thanks @tpoitras I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregate Container Logs]
|Added section on xref:../install_config/aggregate_logging.adoc#fluentd-external-log-aggregator[configuring Fluentd to send logs to an external log aggregator].
%

**Pull request merged**